consensus/consensus_v2.go

```diff
@@ -373,26 +370,27 @@ func (consensus *Consensus) Start(
 						continue
 					}
 					if k != timeoutViewChange {
-						consensus.getLogger().Debug().Msg("[ConsensusMainLoop] Ops Consensus Timeout!!!")
-						consensus.startViewChange(consensus.viewID + 1)
+						consensus.getLogger().Warn().Msg("[ConsensusMainLoop] Ops Consensus Timeout!!!")
+						viewID := consensus.GetCurViewID()
+						consensus.startViewChange(viewID + 1)
 						break
 					} else {
-						consensus.getLogger().Debug().Msg("[ConsensusMainLoop] Ops View Change Timeout!!!")
-						viewID := consensus.current.ViewID()
+						consensus.getLogger().Warn().Msg("[ConsensusMainLoop] Ops View Change Timeout!!!")
+						viewID := consensus.GetViewChangingID()
 						consensus.startViewChange(viewID + 1)
 						break
 					}
 				}
 			case <-consensus.syncReadyChan:
-				consensus.getLogger().Debug().Msg("[ConsensusMainLoop] syncReadyChan")
+				consensus.getLogger().Info().Msg("[ConsensusMainLoop] syncReadyChan")
 				consensus.SetBlockNum(consensus.ChainReader.CurrentHeader().Number().Uint64() + 1)
-				consensus.SetViewID(consensus.ChainReader.CurrentHeader().ViewID().Uint64() + 1)
+				consensus.SetViewIDs(consensus.ChainReader.CurrentHeader().ViewID().Uint64() + 1)
```

Comment:
This is another logic change. the original logic didn't update the view changing id.

Reply:
@LeoHChen